exporter.py

```diff
@@ -62,7 +63,7 @@ def parse_params(logger):
     if options.target:
         os.environ['TARGET'] = options.target
     if not options.target:
-        target = get_targets(options.inventory)[0]
+        target = random.choice(get_targets(options.inventory))
```

Comment:
What's the reason behind randomising the target query order?

Reply:
I stumbled across this by entering some targets with the unreachable vrops mgmt under `vrops_list[0]`. So the exporter was not started with a working destination. Since this option is for testing purposes only and we want to make sure it is not bound to a static target, I randomly selected it.